Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -124,7 +124,9 @@ void GraphicsContextSkia::drawRect(const FloatRect& rect, float borderThickness)
     if (!makeGLContextCurrentIfNeeded())
         return;
 
-    canvas().drawRect(rect, createFillPaint());
+    SkPaint paint = createFillPaint();
+    setFillSource(paint);
```

Comment:
`setFillSource` being a member of `GraphicsContextSkia` sounds like it's setting something on the context. Maybe we could rename it as `setupFillSource` or something like that.